packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdkBridgeExt.kt

```diff
@@ -11,7 +11,8 @@ import com.facebook.react.bridge.WritableNativeMap
 
 internal fun List<*>.toWritableArray(): WritableNativeArray {
     val list = WritableNativeArray()
-    forEach {
+    for (it in iterator()) {
```

Comment:
with such change it is better to use other name than `it`, because `it` is reserved keyword in Kotlin, and in this scope it is just a variable name, which may be confusing.